src/Equinor.ProCoSys.IPO.WebApi/Synchronization/StringToLongConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Equinor.ProCoSys.IPO.WebApi.Synchronization;
+
+public class StringToLongConverter : JsonConverter<long>
+{
+    public override long Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
```

Comment:
How about a couple of unit tests, one that throws an exception and one that passes?